Mithril update 2024-04-10 (#373)
IntersectMBO/cardano-updates: blog/2024-04-10-mithril.md

---
title: Mithril Team Update
slug: 2024-04-10-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They focused on scaling the signature and proof generation for `mainnet`, completed the incremental storage of transactions, activated the signature of the transactions on the `testing-preview` network, and prepared a new network to test the scaling on `mainnet` data. They also kept working on a prototype to decentralize signer registration with the relay and a peer-to-peer (P2P) network. Additionally, the team worked on refactoring the database providers of the aggregator.

Reviewer: ```suggestion
This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They focused on scaling the signature and proof generation for `mainnet`, completed the incremental storage of transactions, activated the signature of the transactions on the `testing-preview` network, and prepared a new network to test the scaling on `mainnet` data. The team also made progress on a prototype for decentralizing signer registration with the relay and a peer-to-peer (P2P) network. Additionally, they worked on refactoring the database providers of the aggregator.
```

> Finally, they worked on a community requested feature to provide in a machine readable format the Cardano node minimum versions supported by the signer and they worked on providing a manual setup for the relay in the SPO user guide.

Reviewer: ```suggestion
Finally, the team delivered a community-requested feature that displays the minimum versions of the Cardano node that the signer supports in a machine-readable format. They also provided a manual setup guide for the relay in the SPO user guide.
```
